MAINTAINERS.md

```diff
@@ -1,2 +1,5 @@
-jmanferdelli@vmware.com, yel@vmware.com, agurajada@vmware.com, johnmanferdelli@hotmail.com
+John Manferdelli    - jmanferdelli@vmware.com, johnmanferdelli@hotmail.com, jlmucbmath@gmail.com
+Ye Li               - yel@vmware.com
+Aditya Gurajada     - agurajada@vmware.com, gapisback@gmail.com
+Radoslav Gerganov   - rgerganov@vmware.com, rgerganov@gmail.com
```

Comment:
@jlmucb @yelvmw @rgerganov -- Please cross-check these mail IDs I'm listing here.

Some of your GitHub check-ins are associated with non-VMware mail IDs. So, I'm listing those in this file.

We need to finalize ASAP.

Reply:
LGTM, thanks